main.c

```diff
@@ -226,6 +230,10 @@ void cleanup_after_vm(supervisor_allocation* heap) {
     free_memory(heap);
     supervisor_move_memory();
 
+    #if CIRCUITPY_CANIO
+    common_hal_canio_reset();
+    #endif
```

Comment:
(Not for this PR.) We should have a reset_common_hal that wraps all of these. Right now they are don't haphazard in reset_port.